src/components/utils/TransactionAmount.vue

```diff
@@ -70,7 +72,10 @@ export default class TransactionAmount extends Vue {
 
   get amountToSelf() {
     // @ts-ignore
-    if (this.transaction.sender === this.address && this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)) {
+    if (
+      this.transaction.sender === this.address &&
+      this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
```

Comment:
linter puts this on two lines, making the `@ts-ignore` fail to apply to the `isMultiPayment` call

Reply:
should be able to resolve this with a `// prettier-ignore` line 🤔

Reply:
```
if (
      this.transaction.sender === this.address &&
      // @ts-ignore
      this.isMultiPayment(this.transaction.type, this.transaction.typeGroup)
) {
```